src/odemis/acq/test/calibration_test.py

```diff
@@ -27,6 +27,8 @@
 import time
 import unittest
 
+import numpy
```

Comment:
This moved to a weird place, I would say I like the old place better up top next to the other external libraries without any enter in between them